testsuite/integration-tests/src/test/java/org/jboss/resteasy/microprofile/test/client/integration/ContainerRestClientProxyTest.java

```diff
@@ -76,7 +75,6 @@ public static WebArchive deployment() throws IOException {
     }
 
     @Inject
-    @RestClient
```

Comment:
You should have test for both injections - with and without `@RestClient` qualifier - may add a second injection point and just assert that none is `null`?

Reply:
Yeah, I do plan on doing that. This was just the simplest case to test with is all.